resources/assets/lib/beatmap-discussions/review-document.ts

```diff
@@ -73,7 +73,7 @@ export function parseFromJson(json: string, discussions: Record<number, BeatmapD
           }],
           discussionId: discussion.id,
           discussionType: discussion.message_type,
-          timestamp: discussion.timestamp,
+          timestamp: discussion.timestamp ? BeatmapDiscussionHelper.formatTimestamp(discussion.timestamp) : null,
```

Comment:
This doesn't work when timestamp is `00:00.000` (because 0 is falsy).

Also the helper returns `undefined`  for null-ish timestamp so the check here can be skipped altogether. The function signature will need an update though. And maybe update the helper to return `null` instead.